package.json

```diff
@@ -36,10 +36,10 @@
     "stage:venia": "yarn venia run start",
     "stats:venia": "yarn venia run build:analyze && yarn venia run stats",
     "storybook:venia": "yarn workspace @magento/venia-ui run storybook",
-    "test": "node --experimental-worker node_modules/.bin/jest",
-    "test:ci": "node --experimental-worker node_modules/.bin/jest --max-workers=3 --json --outputFile=test-results.json",
+    "test": "jest",
+    "test:ci": "jest --no-cache --max-workers=3 --json --outputFile=test-results.json",
```

Comment:
Did you intentionally add the `--no-cache` flag here?

Reply:
No, that was me, sorry. It seemed wise not to rely on cache for the CI tests, so I updated it while I was testing things out.